### PR TITLE
sync wasm-bindgen version between cargo and nix

### PR DESCRIPTION
**Pull request by matzemathics**
`nix flake check` should now work again.

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -27,9 +27,9 @@ rec {
         rust-overlay.overlays.default
         (final: prev: {
           wasm-bindgen-cli = channels.nixpkgs-unstable.wasm-bindgen-cli.override {
-            version = "0.2.92";
-            hash = "sha256-1VwY8vQy7soKEgbki4LD+v259751kKxSxmo/gqE6yV0=";
-            cargoHash = "sha256-aACJ+lYNEU8FFBs158G1/JG8sc6Rq080PeKCMnwdpH0=";
+            version = "0.2.93";
+            hash = "sha256-DDdu5mM3gneraM85pAepBXWn3TMofarVR4NbjMdz3r0=";
+            cargoHash = "sha256-birrg+XABBHHKJxfTKAMSlmTVYLmnmqMDfRnmG6g/YQ=";
           };
```
Comment (mmarx) [inline flake.nix:33]: unstable already has `wasm-bindgen-cli` at 0.2.93, so no need to override here, simply do 
```nix
inherit (channels.nixpkgs-unstable) wasm-bindgen-cli;
```

**Review by mmarx — CHANGES_REQUESTED** [review]

File: flake.nix (hunk 1 above)
Comment (matzemathics) [reply flake.nix:33]: However that solution might then break again in the future, right?

Comment (mmarx) [reply flake.nix:33]: Yeah, that will break once unstable upgrades `wasm-bindgen-cli`, but then we can just bump the bound on the cargo side to fix this.

Comment (matzemathics) [reply flake.nix:33]: So for some reason this version fails on my machine with the same error we got in the ci before, which suggests it is using the wrong version. Do I need to update something?

Comment (mmarx) [reply flake.nix:33]: You'll likely need to run `nix flake update --commit-lock-file`

Comment (matzemathics) [reply flake.nix:33]: `API rate limit exceeded for ***.***.***.***` :sweat_smile:

Comment (mmarx) [reply flake.nix:32]: Haha, last time I saw that, I was traewelling with DB :D

I've pushed the updated lock, could you please check that this now works for you?

**Review by mmarx — APPROVED** [review]

File: flake.nix (hunk 1 above)
Comment (matzemathics) [reply flake.nix]: Seems to work now.

**Pull request merged**